test/helper.rb

```diff
@@ -9,11 +9,30 @@
 
 # Test setup
 MIGRATIONS_ROOT = 'test/migrations'
-Mongoid.configure.connect_to('mongoid_test')
+
+Mongoid.configure.load!("#{__dir__}/mongoid.yml", 'test')
 require 'models/survey_schema'
 
 module TestMongoidRailsMigrations
   class Application < Rails::Application; end
 end
 
 TestMongoidRailsMigrations::Application.load_tasks
+
+# Hide task output
+class Mongoid::Migration
+  def self.puts _
+  end
+end
+
+def invoke(task)
+  Rake.application.tasks.each(&:reenable)
```

Comment:
We can't execute the same rake task multiple times without this, that's sad but default `Rake` behavior.

Reply:
And can't we reenable just the task? like `Rake::Task[task].reenable`?

Reply:
Not really, some tasks are calling other tasks and reenabling only the called task does not reenable the ones called by the caller.

Reply:
Ok